test/verify/check-accounts

```diff
@@ -135,16 +135,30 @@ class TestAccounts(MachineCase):
         m.execute("userdel berta")
         b.wait_not_in_text('#accounts-list', "Berta Bestimmt")
 
-        # Create a locked user
+        # Create a locked user with weak password
         m.execute("sed -i 's/^SHELL=.*$/SHELL=/' /etc/default/useradd")
         b.click('#accounts-create')
         b.wait_visible('#accounts-create-dialog')
         b.set_input_text('#accounts-create-user-name', "jussi")
         b.set_input_text('#accounts-create-real-name', "Jussi Junior")
-        b.set_input_text('#accounts-create-pw1', good_password)
-        b.set_input_text('#accounts-create-pw2', good_password)
```

Comment:
Please keep the test case with the good password as well -- this should not ask twice.

Reply:
We have at least one more place in tests where we set up a good password. But I can keep it here as well

Reply:
That's fine then -- no need to duplicate, it just wasn't clear from the diff.